### PR TITLE
Controllers Branch - Fixing omnitool

### PR DESCRIPTION
**Pull request by jherico**

### CODE REVIEW

File: examples/libraries/omniTool.js [hunk 1]
```diff
@@ -47,21 +49,21 @@ OmniTool = function(side) {
     this.showWand(false);
 
     // Connect to desired events
-    var _this = this;
-    Controller.actionEvent.connect(function(action, state) {
-        _this.onActionEvent(action, state);
+    var that = this;
```
Comment (ZappoMan) [inline examples/libraries/omniTool.js]: why did you change `_this` to `that` -- personally I prefer the `_this` style...

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2476f238d0f5842e8b9208a6614865546e9be592 available at: https://gist.github.com/68ca095291e27c6e9ecf<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6162.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6162.dmg

File: examples/libraries/omniTool.js (hunk 1 above)
Comment (jherico) [inline examples/libraries/omniTool.js]: `that` is apparently the de-facto JS idiom.

Comment (hyperlogic) [inline examples/libraries/omniTool.js]: or `self`

Comment (howard-stearns) [inline examples/libraries/omniTool.js]: I'm pretty confident that every js programmer would recognize 'that', and also recognize 'self' after a moment's pause. They'll probably recognize _that after a moment's pause as well, but they'll think it's weird.  Underscores are usually used for "private properties", There isn't any such thing as a private lexical variable.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5a41e4fd9d283814ea7e8f34dfdc1ec0a7a80330 available at: https://gist.github.com/68ca095291e27c6e9ecf<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6162.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6162.dmg

**Comment by ZappoMan** [comment]
an interesting discussion of that vs self vs other approaches... _this of course is not included...

http://stackoverflow.com/questions/4886632/what-does-var-that-this-mean-in-javascript

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4808f0bddb864bf31759b7735c15b4049658a580 available at: https://gist.github.com/68ca095291e27c6e9ecf<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6162.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6162.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6e3e7b71aec4c98f968bdb3a449e0107cd947200 available at: https://gist.github.com/68ca095291e27c6e9ecf<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6162.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6162.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: f608b2a39fa627acd6720f31c93a85ac6002daca available at: https://gist.github.com/68ca095291e27c6e9ecf<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6162.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6162.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 54c20a8dd7f094c752a469671e66aacb67ea062a available at: https://gist.github.com/68ca095291e27c6e9ecf<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6162.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6162.dmg
